objax/nn/layers.py

```diff
@@ -236,6 +259,13 @@ def __call__(self, x: JaxArray) -> JaxArray:
             y += self.b.value
         return y
 
+    def __repr__(self):
+        args = dict(nin=self.w.value.shape[2] * self.groups, nout=self.w.value.shape[3], k=self.w.value.shape[:2],
```

Comment:
In the init function, when constructor of superclass is called `nin` and `nout` are flipped for `ConvTranspose2D` compared to superclass